cmip6_downscaling/methods/bcsd/tasks.py

```diff
@@ -87,9 +93,31 @@ def fit_and_predict(
         print(f'found existing target: {target}')
         return target
 
-    # TODO
+    # # TODO
+    if run_parameters.variable in ABSOLUTE_VARS:
+        bcsd_model = BcsdTemperature(return_anoms=False)
+    elif run_parameters.variable in RELATIVE_VARS:
+        bcsd_model = BcsdPrecipitation(return_anoms=False)
+
+    pointwise_model = PointWiseDownscaler(model=bcsd_model, dim="time")
+
+    # coarse_obs_rechunked_validated_ds
+    coarse_obs_full_time_ds = xr.open_zarr(coarse_obs_full_time_path)
+    # gcm_train_subset_full_time_ds
+    experiment_train_full_time_ds = xr.open_zarr(experiment_train_full_time_path)
+    # gcm_predict_rechunked_ds
+    experiment_predict_full_time_ds = xr.open_zarr(experiment_predict_full_time_path)
```

Comment:
I don't think these comments add enough to motivate their use here.

```suggestion
    coarse_obs_full_time_ds = xr.open_zarr(coarse_obs_full_time_path)
    experiment_train_full_time_ds = xr.open_zarr(experiment_train_full_time_path)
    experiment_predict_full_time_ds = xr.open_zarr(experiment_predict_full_time_path)
```